### PR TITLE
v14 Rewrite for this src.

### PR DESCRIPTION
**Pull request by awaitlol**
v13 -> v14

### CODE REVIEW

**Comment by awaitlol** [comment]
<@918137699842555964> | <@757296951925538856> maybe pull this for me? :point_right: :point_left:

File: autoRun/updateTicketMessage.js [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
 const config = require('../config.json')
 module.exports = async (client) => {
     let channel = client.channels.cache.get(config.channelID.interactionsChannel);
     let msg = (await channel.messages.fetch({limit: 10}))?.last()
     
     let toSendEmbed = [
-        new Discord.MessageEmbed()
+        new EmbedBuilder()
         .setTitle(`Interactions`)
         .setColor(`BLUE`)
```
Comment (lolollllo) [inline autoRun/updateTicketMessage.js]: `.setColor('Blue')`

File: autoRun/updateTicketMessage.js [hunk 2]
```diff
@@ -1,41 +1,42 @@
...
     let ToSendComponents = [
-        new Discord.MessageActionRow()
+        new ActionRowBuilder()
         .addComponents(
-            new Discord.MessageButton()
+            new ButtonBuilder()
             .setCustomId('CreateTicket')
             .setEmoji('📩')
-            .setStyle('SECONDARY')
+            .setStyle(ButtonStyle.Primary)
```
Comment (lolollllo) [inline autoRun/updateTicketMessage.js:21]: This button had the `Secondary` style

File: autoRun/updateTicketMessage.js [hunk 3]
```diff
@@ -1,41 +1,42 @@
...
             .setEmoji('👨‍💼')
-            .setStyle('SECONDARY')
+            .setStyle(ButtonStyle.Secondary)
             .setDisabled(!config.settings.interactions.staff)
         )
     ]
     
-    if(msg) msg.edit({embeds: toSendEmbed, components: ToSendComponents})
+    if(msg) msg.edit({ embeds: toSendEmbed, components: ToSendComponents})
     else channel.send({embeds: toSendEmbed, components: ToSendComponents})
```
Comment (lolollllo) [inline autoRun/updateTicketMessage.js]:
```diff
- embeds: toSendEmbed
+ embeds: [toSendEmbed]

Comment (lolollllo) [inline autoRun/updateTicketMessage.js]:
```diff
- embeds: toSendEmbed
+ embeds: [toSendEmbed]

File: autoRun/wingsStatus.js [hunk 4]
```diff
@@ -69,7 +69,7 @@ module.exports = async (client) => {
             panellatency = NaN
         })
 
-        let embed = new Discord.MessageEmbed()
+        let embed = new EmbedBuilder()
         .setTitle('ArtiomsHosting Node Status:')
         .setColor(`#677bf9`)
```
Comment (lolollllo) [inline autoRun/wingsStatus.js:74]:
```diff
- .setColor(`#677bf9`)
+ .setColor(0x677bf9)

File: commands/admin/eval.js [hunk 5]
```diff
@@ -22,14 +22,14 @@ module.exports = {
...
 
             if(silent) return
             if(evaled.length > 4000){
                 if(silent) return
                 message.channel.send({
                     embeds:[
-                        new Discord.MessageEmbed()
+                        new EmbedBuilder()
                         .setTitle(`Eval by ${message.author.tag}`)
                     	.setColor(`GREEN`)
```
Comment (lolollllo) [inline commands/admin/eval.js]:
```diff
- .setColor(`GREEN`)
+ .setColor(`Green`)

File: commands/admin/eval.js [hunk 6]
```diff
@@ -45,7 +45,7 @@ module.exports = {
             }else{
                 message.channel.send({
                     embeds:[
-                        new Discord.MessageEmbed()
+                        new EmbedBuilder()
                         .setTitle(`Eval by ${message.author.tag}`)
                     	.setColor(`GREEN`)
```
Comment (lolollllo) [inline commands/admin/eval.js]:
```diff
- .setColor(`GREEN`)
+ .setColor(`Green`)

File: commands/admin/eval.js [hunk 7]
```diff
@@ -56,7 +56,7 @@ module.exports = {
         }catch(err){
             message.channel.send({
                 embeds:[
-                    new Discord.MessageEmbed()
+                    new EmbedBuilder()
                     .setTitle(`:x: | Error`)
                     .setColor(`RED`)
```
Comment (lolollllo) [inline commands/admin/eval.js]:
```diff
- .setColor(`RED`)
+ .setColor(`Red`)

File: commands/info/help.js [hunk 8]
```diff
@@ -1,14 +1,14 @@
...
-            .addField(`**Info:**`, `AH!ping\nAH!invites\nAH!help`, true)
-        ]})
-    }
+const { EmbedBuilder } = require('discord.js');
+module.exports = (client, message, args) => {
+    message.channel.send({
+        embeds:[
+          new EmbedBuilder()
+          .setTitle(`❓ | Need help?`)
+          .setColor(`#677bf9`)
```
Comment (lolollllo) [inline commands/info/help.js:7]:
```diff
- .setColor(`#677bf9`)
+ .setColor(`0x677bf9`)

File: commands/info/ping.js [hunk 9]
```diff
@@ -9,7 +9,7 @@ module.exports = {
             x.edit({
                 content: null,
                 embeds:[
-                    new Discord.MessageEmbed()
+                    new EmbedBuilder()
                     .setTitle(`🏓 | Bot ping`)
                     .setColor(`${responce > 300 ? responce > 1000 ? "RED" : "YELLOW" : "GREEN"}`)
```
Comment (lolollllo) [inline commands/info/ping.js]:
```diff
- .setColor(`${responce > 300 ? responce > 1000 ? "RED" : "YELLOW" : "GREEN"}`)
+ .setColor(`${responce > 300 ? responce > 1000 ? "Red" : "Yellow" : "Green"}`)

File: commands/info/snipe.js [hunk 10]
```diff
@@ -20,12 +20,12 @@ module.exports = {
 
         let snipedMessage = snipe[number];
 
-        const embed = new Discord.MessageEmbed()
+        const embed = new EmbedBuilder()
         embed.setTitle(`Sniping...`)
         embed.setColor(`BLUE`)
```
Comment (lolollllo) [inline commands/info/snipe.js]:
```diff
- embed.setColor(`BLUE`)
+ embed.setColor(`Blue`)

File: commands/info/snipe.js [hunk 11]
```diff
@@ -20,12 +20,12 @@ module.exports = {
...
-        snipedMessage.messageContent ? embed.addField(`Message Content`, `${snipedMessage.messageContent}`) : null
+        snipedMessage.messageContent ? embed.addField({ name: `Message Content`, value:`${snipedMessage.messageContent}`) : null
```
Comment (lolollllo) [inline commands/info/snipe.js]: `addFields()` not `addField()`

File: commands/info/snipe.js [hunk 12]
```diff
@@ -20,12 +20,12 @@ module.exports = {
...
         snipedMessage.messageImages?.size !== 0 ? embed.setImage(snipedMessage.messageImages?.first()?.proxyURL) : null
-        embed.setFooter({ text: `${snipedMessage.member.user.tag} (${snipedMessage.member.user.id})`, iconURL: snipedMessage?.member?.user?.displayAvatarURL()});
+        embed.setFooter({ text: ${snipedMessage.member.user.tag} (${snipedMessage.member.user.id}), iconURL: snipedMessage?.member?.user?.displayAvatarURL()});
```
Comment (lolollllo) [inline commands/info/snipe.js]: Missing back quotes

File: commands/server/count.js [hunk 13]
```diff
@@ -13,9 +13,9 @@ module.exports = async (client, message, args) => {
 
     message.channel.send({
         embeds:[
-            new Discord.MessageEmbed()
+            new EmbedBuilder()
             .setTitle(`${user.username}'s Server Count`)
-            .setColor(`#677bf9`)
+            .setColor(``#677bf9`)
```
Comment (lolollllo) [inline commands/server/count.js]: Extra back quote + 
```diff
- .setColor(`#677bf9`)
+ .setColor(`0x677bf9`)

File: commands/server/create.js [hunk 14]
```diff
@@ -9,23 +10,25 @@ module.exports = async (client, message, args) => {
...
         .setLabel("Panel")
         
-        const row = new Discord.MessageActionRow()
+        const row = new ActionRowBuilder()
         .addComponents([panelButton])
         
-        const noTypeListed = new Discord.MessageEmbed() 
+        const noTypeListed = new EmbedBuilder() 
             .setTitle(`Types of servers you can create: `)
             .setColor(`#677bf9`)
```
Comment (lolollllo) [inline commands/server/create.js:23]:
```diff
- .setColor(`#677bf9`)
+ .setColor(`0x677bf9`)

File: commands/server/create.js [hunk 15]
```diff
@@ -70,9 +73,9 @@ module.exports = async (client, message, args) => {
         msg.edit({
             content: null,
             embeds:[
-                new Discord.MessageEmbed()
+                new EmbedBuilder()
                 .setColor(`GREEN`)
```
Comment (lolollllo) [inline commands/server/create.js]:
```diff
- .setColor(`GREEN`)
+ .setColor(`Green`)

File: commands/server/create.js [hunk 16]
```diff
@@ -89,45 +92,52 @@ module.exports = async (client, message, args) => {
             msg.edit({
                 content: null,
                 embeds:[
-                    new Discord.MessageEmbed()
+                    new EmbedBuilder()
                     .setColor('RED')
```
Comment (lolollllo) [inline commands/server/create.js]:
```diff
- .setColor(`RED`)
+ .setColor(`Red`)

File: commands/server/create.js [hunk 17]
```diff
@@ -89,45 +92,52 @@ module.exports = async (client, message, args) => {
...
+    )
                 ]
             })
         }else if (error == "Error: Request failed with status code 504") {
             msg.edit({
                 content: null,
                 embeds:[
-                    new Discord.MessageEmbed()
+                    new EmbedBuilder()
                     .setColor('RED')
```
Comment (lolollllo) [inline commands/server/create.js]:
```diff
- .setColor(`RED`)
+ .setColor(`Red`)

File: commands/server/create.js [hunk 18]
```diff
@@ -89,45 +92,52 @@ module.exports = async (client, message, args) => {
...
+    )
                 ]
             })
         }else if (error == "Error: Request failed with status code 429") {
             msg.edit({
                 content: null,
                 embeds:[
-                    new Discord.MessageEmbed()
+                    new EmbedBuilder()
                     .setColor('RED')
```
Comment (lolollllo) [inline commands/server/create.js]:
```diff
- .setColor(`RED`)
+ .setColor(`Red`)

File: commands/server/create.js [hunk 19]
```diff
@@ -89,45 +92,52 @@ module.exports = async (client, message, args) => {
...
+		{ name: '❌ Server creation failed', value: `Uh oh, This shouldn\'t happen, Try again in a minute or two.` }
                     ]
             })
         }else if (error == "Error: Request failed with status code 429") {
             msg.edit({
                 content: null,
                 embeds:[
-                    new Discord.MessageEmbed()
+                    new EmbedBuilder()
                     .setColor('RED')
```
Comment (lolollllo) [inline commands/server/create.js]:
```diff
- .setColor(`RED`)
+ .setColor(`Red`)

File: commands/server/create.js [hunk 20]
```diff
@@ -89,45 +92,52 @@ module.exports = async (client, message, args) => {
...
+		{ name: '❌ Server creation failed', value: `Uh oh, This shouldn\'t happen, Try again in a minute or two.` }
                     ]
             })
         }else {
             msg.edit({
                 content: null,
                 embeds:[
-                    new Discord.MessageEmbed()
+                    new EmbedBuilder()
                     .setColor('RED')
```
Comment (lolollllo) [inline commands/server/create.js]:
```diff
- .setColor(`RED`)
+ .setColor(`Red`)

**Review by lolollllo — CHANGES_REQUESTED** [review]

**Comment by lolollllo** [comment]
There's more which I'll review later, also would you please run prettier on this

**Comment by awaitlol** [comment]
Sure. I'll also double check some code.

**Comment by awaitlol** [comment]
Also please check https://github.com/TheCtkHoster/ArtiomsHosting-Bot-v14/blob/main/commands/info/snipe.js (snipe.js) the embed because I am not too sure if it will work.

**Comment by awaitlol** [comment]
Updated everything, (including events). Should work now :)

**Comment by awaitlol** [comment]
image

Didn't update some files because they already included HEX code in them.

**Comment by lolollllo** [comment]
> Didn't update some files because they already included HEX code in them.
setColor() takes a number though (or a string if your using the names of the colors) 

https://discord.js.org/#/docs/discord.js/main/typedef/ColorResolvable

**Comment by awaitlol** [comment]
Hex still works tho so yeah.

**Comment by lolollllo** [comment]
> Hex still works tho so yeah.
oh kay

**Comment by awaitlol** [comment]
I'll recheck the code again for `MessageEmbed` and other things where I didn't replace the v13 code with v14.

**Comment by awaitlol** [comment]
https://github.com/TheCtkHoster/ArtiomsHosting-Bot-v14 (code link btw)

**Comment by awaitlol** [comment]
Knew it, I forgor to edit https://github.com/TheCtkHoster/ArtiomsHosting-Bot-v14/blob/main/autoRun/updateTicketMessage.js (doing it now)

**Comment by DEV-DIBSTER** [comment]
So I just checked through the current running code, it seems that the code that's currently running is vastly different from the current V13 code located within this repository. 

I'll see if we're still looking into a V14 update for this Discord bot, will still need to be decided.

**Comment by awaitlol** [comment]
> So I just checked through the current running code, it seems that the code that's currently running is vastly different from the current V13 code located within this repository.
> 
> I'll see if we're still looking into a V14 update for this Discord bot, will still need to be decided.

Can you show me where there is v13 code in this repo so I can fix them?

**Pull request closed without merging**